driver/pelican/interface.md

```diff
@@ -0,0 +1,40 @@
+### Pelican Schedule Interface Outline
+
+##### Context
+
+The following structs define the way users interact with the Scheduling interface of the Pelican thermostats. These structs are used to define the weekly schedule and will be interpretted by the thermSchedule.go code to retrieve/view in addition to making changes to the existing schedule for individual thermostats within a particular site. These structs are public and accessible to anyone who subscribes to the allotted endpoint or publishes to the assigned signal.
+
+##### Schedule Structs
+
+// Struct mapping each day of the week to its daily schedule <br>
```

Comment:
Can you clean up the struct definitions by removing the `&nbsp;` and `<br>` elements from the text, and putting the code inside a monospace code environment? It should look something like this (https://help.github.com/articles/creating-and-highlighting-code-blocks/)

Reply:
No problem! Just changed it up, I knew this but totally forgot about it. Thanks for the reminder haha.